<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSeriesWindow</class>
 <widget class="QDialog" name="AddSeriesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>250</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Add Series</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>231</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Year</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Country</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Duration</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Rating</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Seasons</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Episodes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Last watched
Season</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Last watched
Episode</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="AddButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>560</y>
        <width>251</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CancelButton">
      <property name="geometry">
       <rect>
        <x>272</x>
        <y>560</y>
        <width>231</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="enter_lSeason">
      <property name="geometry">
       <rect>
        <x>271</x>
        <y>330</y>
        <width>205</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="enter_rating">
      <property name="geometry">
       <rect>
        <x>271</x>
        <y>210</y>
        <width>205</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="enter_lEpisode">
      <property name="geometry">
       <rect>
        <x>271</x>
        <y>390</y>
        <width>205</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="enter_episodes">
      <property name="geometry">
       <rect>
        <x>271</x>
        <y>290</y>
        <width>205</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="enter_title">
      <property name="geometry">
       <rect>
        <x>271</x>
        <y>50</y>
        <width>205</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="enter_year">
      <property name="geometry">
       <rect>
        <x>271</x>
        <y>90</y>
        <width>205</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="enter_duration">
      <property name="geometry">
       <rect>
        <x>271</x>
        <y>170</y>
        <width>205</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="enter_country">
      <property name="geometry">
       <rect>
        <x>271</x>
        <y>130</y>
        <width>205</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="enter_seasons">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>250</y>
        <width>205</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>107</width>
        <height>65</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="CheckEnded">
         <property name="text">
          <string>Ended</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckWatched">
         <property name="text">
          <string>Watched</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
